### DISCUSSION TITLE
Change username option is not present in github > settings> accounts

### DISCUSSION THREAD

**Discussion by vinodakumarsr** [community/community#15761] [Other Feature Feedback, Questions, & Ideas]
Change username is not available in latest version of github could you please check it with high priority?

**Comment by crystalcommunication** [comment]
This option? 
![image](https://user-images.githubusercontent.com/71373843/166125419-617fa549-5e20-43f9-97e6-9c03b7808d0b.png)

**Reply by LosAhuertos** [reply]
> Этот вариант? ![изображение](https://user-images.githubusercontent.com/71373843/166125419-617fa549-5e20-43f9-97e6-9c03b7808d0b.png)

Полина пора, а без трусиков
